### PR TITLE
fix(platform): validate firewall

### PR DESCRIPTION
**Pull request by Huimintai**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by leoryu — APPROVED** [review]

File: pkg/util/ssh/os.go [hunk 1]
```diff
@@ -177,3 +177,32 @@ func ReservePorts(s Interface, ports []int) error {
 	}
 	return nil
 }
+
+func FirewallEnabled(s Interface) (enabled bool, err error) {
```
Comment (leoryu) [inline pkg/util/ssh/os.go:181]: check through diff os

Comment (leoryu) [reply pkg/util/ssh/os.go:181]: os get with OSVersion func

File: pkg/util/ssh/os.go [hunk 2]
```diff
@@ -177,3 +177,32 @@ func ReservePorts(s Interface, ports []int) error {
...
+		return false, err
+	}
+	res1 := strings.TrimSpace(stdoutexec)
+	if res1 == "active" {
+		return true, nil
+	}
+	return false, nil
+}
+
+func SelinuxEnabled(s Interface) (enabled bool, err error) {
```
Comment (leoryu) [inline pkg/util/ssh/os.go:201]: same with FirewallEnabled

File: pkg/util/ssh/os.go [hunk 3]
```diff
@@ -177,3 +177,38 @@ func ReservePorts(s Interface, ports []int) error {
...
 	return nil
 }
+
+func FirewallEnabled(s Interface) (enabled bool, err error) {
+	ostype, err := OSVersion(s)
+	if err != nil {
+		return false, err
+	}
+	switch {
+	case strings.Contains(ostype, "tencentos"):
```
Comment (leoryu) [inline pkg/util/ssh/os.go:187]: please use hasprefix func ti check os

File: pkg/util/ssh/os.go [hunk 4]
```diff
@@ -177,3 +177,38 @@ func ReservePorts(s Interface, ports []int) error {
...
+		return res == "1", nil
+	case strings.Contains(ostype, "ubuntu"):
+		stdout, _, exit, err := s.Exec("ufw status | awk '{print $2}'")
+		if err != nil || exit != 0 {
+			return false, err
+		}
+		res := strings.TrimSpace(stdout)
+		return res == "active", nil
+	default:
+		return false, fmt.Errorf("not supported os type: %s", ostype)
```
Comment (leoryu) [inline pkg/util/ssh/os.go:202]: default use tencentos aciton

**Review by huxiaoliang — APPROVED** [review]

**Pull request merged**